app/views/users/index.html.erb

```diff
@@ -1,2 +1,44 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
+
+<section>
+  <% @users.each do |user| %>
+  <div>
+    <%= image_tag user.photo %> 
+    <h2><%= link_to user.name, user_path(user) %></h2>
+    <p>Number of post<%= user.posts_counter %></p>
+  </div>
+  <% end %>
+</section>
+
+<%# 
+first_user = User.create(name: 'Tom', photo: 'https://img.freepik.com/free-photo/bicycle-basket-with-flowers-books-high-angle_23-2149739959.jpg', bio: 'Teacher from Mexico.')
+second_user = User.create(name: 'Lilly', photo: 'https://img.freepik.com/free-photo/bicycle-basket-with-flowers-books-high-angle_23-2149739959.jpg', bio: 'Teacher from Poland.')
+third_user = User.create(name: 'Kevin', photo: 'https://img.freepik.com/free-photo/bicycle-basket-with-flowers-books-high-angle_23-2149739959.jpg', bio: 'Developer from Ghana.')
+fourth_user = User.create(name: 'James', photo: 'https://img.freepik.com/free-photo/bicycle-basket-with-flowers-books-high-angle_23-2149739959.jpg', bio: 'farmer from Pakistan.')
+
+
+
```

Comment:
- [ ] Kindly remove all pieces of commented codes from your codebase. You mentioned in your PR that you will remove it, but I don't think it is necessary to keep it in the first place.